addons/ccm-openstack/ccm-openstack.yaml

```diff
@@ -199,19 +218,17 @@ spec:
 {{ end }}
           env:
             - name: CLOUD_CONFIG
-              value: /etc/config/cloud-config
+              value: /etc/config/cloud.conf
```

Comment:
Something is fucky, kubeone automatically creates `cloud-config`, which is later mounted into the pod.

Reply:
Even in the old chart version, the file name itself was never templated upstream (https://github.com/kubernetes/cloud-provider-openstack/blob/openstack-cloud-controller-manager-2.28.3/charts/openstack-cloud-controller-manager/templates/daemonset.yaml#L98). I presume this was always (like in KKP) adjusted by hand.

Reply:
I've added kustomize code to fix the env value.